util/quality-measures-additional-info.json

```diff
@@ -0,0 +1,1217 @@
+[
+  {
```

Comment:
Do the descriptions need to be in a separate file? Also maybe the file name could be more explicit `quality-measures-strata` or something?